### PR TITLE
PR for #90 - Add missing Operators.rb

### PR DESCRIPTION
**Pull request by chess-equality**
#90

### CODE REVIEW

File: src/main/groovy/com/codebrig/arthur/observe/structure/filter/operator/relational/RelationalOperatorFilter.groovy [hunk 1]
```diff
@@ -58,4 +61,52 @@ class RelationalOperatorFilter extends StructureFilter<RelationalOperatorFilter,
         }
         return false
     }
+
+    static SourceNode getLeftOperand(SourceNode node) {
```
Comment (BFergerson) [inline src/main/groovy/com/codebrig/arthur/observe/structure/filter/operator/relational/RelationalOperatorFilter.groovy]: It looks like there will be a conflict here and https://github.com/CodeBrig/Arthur/blob/c3fdd43e3cc4c758bee987ec140a8f51ec03251b/src/main/groovy/com/codebrig/arthur/observe/structure/filter/operator/relational/RelationalOperatorFilter.groovy#L66. Please update them so they will not conflict on merge.

Comment (BFergerson) [reply src/main/groovy/com/codebrig/arthur/observe/structure/filter/operator/relational/RelationalOperatorFilter.groovy]: By the conflict, I'm talking about the `getSendOperatorLeft` vs `getPropOperand1`.

Comment (chess-equality) [reply src/main/groovy/com/codebrig/arthur/observe/structure/filter/operator/relational/RelationalOperatorFilter.groovy]: @BFergerson These two method groups should actually be added and merged together in `RelationalOperatorFilter.groovy`, but they're in different branches/issues:

- `getPropOperand1` / `getPropOperand1` (#40)
- `getSendOperatorLeft` / `getSendOperatorRight` (#90)

Should I added them in `RelationalOperatorFilter.groovy` for both branches, or what do you suggest?

Comment (BFergerson) [reply src/main/groovy/com/codebrig/arthur/observe/structure/filter/operator/relational/RelationalOperatorFilter.groovy]: @chess-equality, I understand that both `getPropOperand1/getSendOperatorLeft` (etc) are needed but the implementing function `getLeftOperand` is different between the two branches.

It looks like `getLeftOperand` will need to switch based off language. As for the merging, you can copy + paste the same code in both branches.

I hope that makes sense.

Comment (chess-equality) [reply src/main/groovy/com/codebrig/arthur/observe/structure/filter/operator/relational/RelationalOperatorFilter.groovy]: @BFergerson Please see my commits:

[[#40] Added getSendOperatorLeft / getSendOperatorRight](https://github.com/CodeBrig/Arthur/pull/86/commits/722547f02cbce050590d0d18ab29124c39ef829f)

[[#90] Added getPropOperand1 / getPropOperand2](https://github.com/CodeBrig/Arthur/pull/93/commits/1ce63d18e67b0d0f2782adbd716bfb9b99c15f34)

C++ branch is not yet merged so I did not use language switches here.

**Comment by BFergerson** [comment]
@chess-equality, please resolve the merge conflicts.

**Comment by chess-equality** [comment]
@BFergerson It's resolved and merged.

**Comment by BFergerson** [comment]
@chess-equality, got some more merge conflicts. Please resolve.

**Comment by chess-equality** [comment]
@BFergerson It's resolved and merged.

**Pull request merged**